Update Awesome Holo

f13end opened the pull request.

f13end committed:
Awesome Holo

f13end committed:
Update AWESOME.md

f13end committed:
Create Başlıksız-1.png

f13end committed:
TEST LOGO

f13end committed:
Update logo.png

f13end committed:
Delete logo.png

f13end committed:
Create logo.png

f13end committed:
Update AWESOME.md

f13end committed:
to be continued..

f13end committed:
update

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Merge branch 'master' into master

f13end committed:
Create AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
update logo

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update LogoTR.png

f13end committed:
Update LogoTR.png

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Awesome Turkish and Awesome updated.

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update Awesome

f13end committed:
Test

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME.md

f13end committed:
Awesome

f13end committed:
Update

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Updated Awesome..

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME.md

d3sbar commented:
@dhtnetwork Can someone do some review of this pull request re: editing? At least the official links.

Connoropolous commented on AWESOME.md:
```diff
@@ -36,3 +41,180 @@
+
+**For topic-organized, detailed discussion, join us on Mattermost (it’s like Slack):**
+
+- https://chat.holochain.net
```
chat.holochain.org only now :)

Connoropolous commented on AWESOME.md:
```diff
@@ -36,3 +41,180 @@
+ - Hackstage: Event mangagement
+ - MyData private data sharing:
+ - Unmute / Medium / P2P Blognet:
+ - P2P Library:
```
Perhaps you mean 'MailBoox' by Viktor Zaunders?
https://github.com/zaunders/Mailboox-Rust

f13end committed:
Update AWESOME.md

f13end committed:
Orphaned apps changed.

Deleted. 

 - Hackstage: Event mangagement
 - MyData private data sharing:
 - Unmute / Medium / P2P Blognet:
 - P2P Library:

Added.

[Mailboox - A distributed liberary](https://github.com/zaunders/Mailboox-Rust)

f13end commented:
I think everything is great.

Connoropolous commented on AWESOME.md:
```diff
@@ -36,3 +41,177 @@
+
+#### Developer Kit Resources
+
+- [Crate HDK](https://developer.holochain.org/api/0.0.2/hdk/index.html)
```
maybe go for 0.0.3 version for now, since it's the latest?
```suggestion
- [Crate HDK](https://developer.holochain.org/api/0.0.3/hdk/index.html)
```

Connoropolous committed:
Update AWESOME.md

Co-Authored-By: f13end <elitearmyleader@gmail.com>

f13end commented:
@Connoropolous Added your suggestion.

f13end committed:
Update AWESOME.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Update AWESOME_Turkish.md

f13end committed:
Change Holochain 0.0.4 links

f13end committed:
Update AWESOME.md
